index.html

```diff
@@ -2,316 +2,349 @@
-						<img class="vector" src="images/right-arrow.png" alt="arrow right" />
-					</div>
-				</section>
-			</div>
-		</section>
+                        <li class="Job-title_and_year on-small">
+                            Back End Dev
+                        </li>
+                        <li class="Job-title_and_year on-large">
+                            Lead Developer
+                        </li>
+                        <li class="Job-title_and_year">
+                            2022
+                        </li>
+                    </ul>
+                </section>
+                <p class="notice font-style">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="programming-languages font-style">
+                    <li>html</li>
+                    <li class="on-large">Ruby On Rails</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <button class="project-button button" type="button">See Project</button>
+            </section>
+        </section>
+        <section class="card third-card">
+            <img class="snapshoot-style on-small" src="images/Snapshoot Portfolio3.svg" alt="Snapshoot portfolio image">
+            <img class="snapshoot-style on-large" src="images/Snapshoot Portfolio5.svg" alt="Snapshoot portfolio image">
+            <section>
+                <h1 class="card-header font-style">
+                    Tonic
+                </h1>
+                <section class="card-description">
+                    <ul class="font-style">
+                        <li class="card-list-title"> CANOPY</li>
+                        <li class="Job-title_and_year">
+                            Back End Dev
+                        </li>
+                        <li class="Job-title_and_year">
+                            2022
+                        </li>
+                    </ul>
+                </section>
+                <p class="notice font-style">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="programming-languages font-style">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <button class="project-button button" type="button">See Project</button>
+            </section>
+        </section>
+        <section class="card fourth-card">
+            <img class="snapshoot-style" src="images/Snapshoot Portfolio4.svg" alt="Snapshoot portfolio image">
+            <section>
+                <h1 class="card-header font-style on-small">
+                    Multi-Post <br> Stories
+                </h1>
+                <h1 class="card-header font-style on-large">
+                    Multi-Post Stories
+                </h1>

```

Comment:
- [ ] I would kindly suggest you to use a `hx` html rules   for proper semantic reason kindly use `h3 tag` instead of `h1 tag` becouse you have use the `h1` ones in your heading.kindly fix this for semantic reason.